agenta-cli/agenta/sdk/tracing/spans.py

```diff
@@ -0,0 +1,92 @@
+from json import loads, dumps
```

Comment:
why do we provider this wrapper around the otel SDK?

Reply:
Because we handle (de-)serialization of all attributes (standard and non-standard) for the user.
Just pushed an update, here